src/models/operation.ts

```diff
@@ -1,14 +1,18 @@
 import type { BaseModel } from './base';
-import type { MessagesInterface } from './messages';
 import type { OperationTraitsInterface } from './operation-traits';
 import type { OperationTraitInterface } from './operation-trait';
+import type { ChannelsInterface } from './channels';
 import type { ServersInterface } from './servers';
+import type { MessagesInterface } from './messages';
 
 export type OperationAction = 'send' | 'receive' | 'publish' | 'subscribe';
 
 export interface OperationInterface extends BaseModel, OperationTraitInterface {
-  action(): OperationAction;
+  action(): OperationAction | undefined;
```

Comment:
Can operations really have undefined action? 🤔

Reply:
Yeah, right, I will fix that - it's a leftover when we could define action in Operation Trait.

Reply:
Ok, done :) Could you check again?